akka-sample-persistence-dc-scala/build.sbt

```diff
@@ -33,4 +33,7 @@ libraryDependencies ++= Seq(
   "org.scalatest" %% "scalatest" % "3.0.7" % "test"
 )
```

Comment:
application.conf can be simplified slightly, artery is default

Reply:
also update logback.xml, see example in https://github.com/akka/akka-samples/pull/135

Reply:
Added in the async appender. Hadn't thought about that for STDOUT